doc/index.md

```diff
@@ -98,7 +98,10 @@ You can also access the latest version (or a specific version) via the CDN. Exam
 
 <!-- then access the sdk via the window object -->
 <script>
-  const sdk = new window.GenesysCloudWebrtcSdk(...); // same usage as above
+  /* in v7, all files are exported under the `window.GenesysCloudWebrtcSdk` */
+  const sdk = new window.GenesysCloudWebrtcSdk.GenesysCloudWebrtcSdk({...}); // same usage as above
```

Comment:
Why are we doing this?

Reply:
I changed this in v7. We were only exporting `window.GenesysCloudWebrtcSdk` as the `client class`. So if you wanted anything else exported in `index.ts`, you were up a creek. I can change it back. I know this looks ugly, but it was unfortunate to only export the client class and nothing else (imo). Up for debate, though.

Reply:
ok. I guess i'm ok with this then.